### ISSUE TITLE
dua --help suggests subcommand comes last

### ISSUE THREAD

**Issue by juhp** [Byron/dua-cli#71]
The help output suggests that subcommands come last:
```
$ ./.cargo/bin/dua -h
 Sat 11:48:24 
dua 2.10.3
A tool to learn about disk usage, fast!

USAGE:
    dua [FLAGS] [OPTIONS] [input]... [SUBCOMMAND]
: 
```
but actually that is not true, the subcommands come before the input dirs.
(Help for the subcommands correctly reflects that.)

**Comment by Byron** [comment]
Thanks for letting me know! I believe this is an issue with the underlying commandline argument parser, clap, which comes in via structupt.

I will see if this is fixed with clap 3.0.0 beta.2, and post an issue there if it is not.

**Comment by Byron** [comment]
Fixed in v2.10.5.
